odbc/src/api.rs

```diff
@@ -1115,12 +1143,56 @@ pub extern "C" fn SQLSetEnvAttr(
 
 #[no_mangle]
 pub extern "C" fn SQLSetEnvAttrW(
-    _environment_handle: HEnv,
-    _attribute: EnvironmentAttribute,
-    _value: Pointer,
+    environment_handle: HEnv,
+    attribute: EnvironmentAttribute,
+    value: Pointer,
     _string_length: Integer,
 ) -> SqlReturn {
-    unimplemented!()
+    // let mut env_handle = environment_handle as *mut MongoHandle;
+    let env_handle = MongoHandleRef::from(environment_handle);
+    env_handle.clear_diagnostics();
+    match env_handle.as_env() {
+        None => SqlReturn::INVALID_HANDLE,
+        Some(env) => match attribute {
+            EnvironmentAttribute::OdbcVersion => match value as Integer {
+                380 => {
```

Comment:
Ideally, I'd be able to match on the `odbc-sys` enums (e.g. `AttrOdbcVersion`), but those enums don't derive `FromPrimitive`.

Reply:
oh :(
Could you do something like getting a int value from the enum though? So that you don't have to hardcode the possible values and we have less room for mistakes (and if there mistake it will be from odbc-sys)?

Reply:
I could do `if value as Integer == AttrOdbcVersion::Odbc3_80 as Integer { ... } else { ... }`